### ISSUE TITLE
Can anyone help me compile EmulationStation independently?

### ISSUE THREAD

**Issue by SX-111** [RetroPie/EmulationStation#478]
I would like to compile EmulationStation independently, without installing RetroPie.

In the official page of EmulationStation there is a guide to do it, I followed the instructions and it does not work. 

I leave the link to the independent installation guide: [https://emulationstation.org/gettingstarted.html#install_rpi_standalone](url)

I use Raspbian Stretch Lite 9 (without desktop), which can be downloaded at the following link: [https://www.raspberrypi.org/downloads/raspbian/](url)

**If you need more information or want to know what are the errors I had, please let me know by the comments.**
_**If you can help me, I thank you very much!!**_

**Comment by SX-111** [comment]
So can I ask in the RetroPie forum about how EmulationStation is installed without installing RetroPie?

At first I did not want to ask in the RetroPie forum because I thought they would reject me or something like that.

I will consult in the RetroPie forum, if you authorize me.

**Comment by joolswills** [comment]
First try and follow the instructions I linked. Also, why not just install it using the retropie script if you are unable to compile it manually.

**Comment by SX-111** [comment]
I would like to learn how to do it manually. But I didn't know that RetroPie-Setup could do that.

**Comment by SX-111** [comment]
One more question . . .

Can I build EmulationStation using this fork or do you not authorize it?

**Comment by joolswills** [comment]
Yes. and use the forum for further questions as I have requested.

**Comment by SX-111** [comment]
**Millions of thanks, friend!!!**  :-) 

I'm going to try, friend. If it does not work, I will not hesitate to consult the RetroPie forum.

Thank you **@joolswills** for dedicating your time and sorry for the inconvenience!!
